packages/graphql/src/translate/create-aggregate-where-and-params.ts

```diff
@@ -0,0 +1,339 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Node, Relationship } from "../classes";
+import { RelationField, Context, BaseField } from "../types";
+
+type Operator = "=" | "<" | "<=" | ">" | ">=";
+
+const logicalOperators = ["EQUAL", "GT", "GTE", "LT", "LTE"];
+
+const aggregationOperators = ["SHORTEST", "LONGEST", "MIN", "MAX"];
+
+function createOperator(input): Operator {
+    let operator: Operator = "=";
+
+    switch (input) {
+        case "LT":
+            operator = "<";
+            break;
+        case "LTE":
+            operator = "<=";
+            break;
+        case "GT":
+            operator = ">";
+            break;
+        case "GTE":
+            operator = ">=";
+            break;
+        default:
+            operator = "=";
+            break;
+    }
+
+    return operator;
+}
+
+function aggregate({
+    inputValue,
+    nodeOrRelationship,
+    chainStr,
+    variable,
+}: {
+    inputValue: any;
+    nodeOrRelationship: Node | Relationship;
+    chainStr: string;
+    variable: string;
+}): { aggregations: string[]; params: any; withStrs: string[] } {
+    const aggregations: string[] = [];
+    let withStrs: string[] = [];
+    let params = {};
+
+    Object.entries(inputValue).forEach((e) => {
```

Comment:
Using destructuring in e may give a slightly more readable variables and could be useful to provide some context on what e[0] and e[1] is
```suggestion
    Object.entries(inputValue).forEach(([e1, e2]) => {
```

Reply:
Yes, a better description on what `e[0]` etc is would help when reading the code in the future.